workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Dashboard Basics
+---
+
+The Dashboard Builder allows you to generate new dashboards in undere a minute, integrate custom widgets into existing dashboards, or effortlessly clone default dashboards for further refinement. Drill into any data that you port to Middleware with advanced filtering options by date range and metric value.
+
+# How Dashboards Work
+
+The Dashboard Builder is a Graphical User Interface (GUI) based approach that enables users to create complex query logic without the added stress of learning a new query language. The metrics made available to users when constructing dashboards are dynamically generated based on the data that users port to Middleware. 
+
+# Create New Dashboard
+
+Create a new dashboard by selecting a data source to start your dashboard with:
+
+![](https://lh7-us.googleusercontent.com/awzI7eStfpqKWy78cOJdW-8pMyrVL6lrocPdQnmVlqrEtGh6QFjABrm-GoaCyG3URvGjdRnm_Ukzw4ZrbY6U-Li9QlY-eWKJ3dFzTzO_fb6IT48onLDEUZxaJ5EPDKJwWm6WLs_hzjmGJJTsCvq6AgQ)
+
+Select the metrics you would like to include as widgets in your dashboard:
+
+![](https://lh7-us.googleusercontent.com/MYBkwh9gWohgGbuF9CScQlUi5RU-0w0J8AVXTY3_MqEnidFnqTzGgXSLMy4iVcpV3_aRgrA3fCnxRZU53vGuL5Lp1RUL7ljVXmRdJfhbqptdDmcATY9LDiTDG715qWoX5ABfvTFuFVgmqArf6UC1i1I)
+
+Once you have added widgets to your dashboard, you will be taken directly to your dashboard:
+
+<Note> Data will appear momentarily after you have created your new dashboard </Note>
+
+![](https://lh7-us.googleusercontent.com/rg4LfJetovyC5UMrgE7H3aN3UUuK0K2kbbHp5LYgGHLs12M-48jW3OtN7RlPA0sJnTznOpcdtNFqghehlroWY7lhsw1eQsE1ZSKXSqf8e-EUNX99RzcCr24ScSxFiXhp27NvLokBGwV9PMuKWk8jwTY)
+
+# Create New Widget
```

Comment:
Given this page is a dashboard basics page let's take the "step x" out of it, as we're not guiding a user through the exact steps, but rather explaining all topics within the widget builder. This will also help us group things more explicitly, e.g. "Create Query" has good sub-sections, and we could do similar for "Display Preferences" and "Advanced" (which should be it's own section). We are going to continually add things to these sections so that sets us up for growth.